### ISSUE TITLE
500 Error when accessing diagnostics for page

### ISSUE THREAD

**Issue by webrgp** [putyourlightson/craft-blitz#723]
### Bug Report

Confirmed the issue when accessing page that has `elementQueryType` of type `craft\\elements\\Category`. I've tracked down the issue and adding parenthesis to [this line](https://github.com/putyourlightson/craft-blitz/blob/bcd40cb43d92850627d6b2419b5341413bb2f53f/src/templates/_utilities/diagnostics/pages/page.twig#L85) seems to fix it.

```twig
{# add parenthesis after displayName #}
{{ create(elementQueryType.type).displayName() }} Queries
```

### Diagnostics Report

Craft: 4.11.5
Blitz: 4.23.2

**Comment by webrgp** [comment]
Same in `src/templates/_utilities/diagnostics/pages/element-query-type.twig`

**Comment by bencroker** [comment]
Hmm, that’s strange, `create(elementQueryType.type).displayName` should work just fine. Can you please check what the exact exception message is in the logs?

**Comment by webrgp** [comment]
Sure. The PHP version 8.1.30. I see the same issue on other templates as I keep adding parenthesis to
`displayName` and clicking throught.

```
2024-10-22 09:23:50 2024-10-22 08:23:50 [web.ERROR] [yii\base\InvalidConfigException] yii\base\InvalidConfigException: Category is missing its group ID in /app/vendor/craftcms/cms/src/elements/Category.php:737
2024-10-22 09:23:50 Stack trace:
2024-10-22 09:23:50 #0 /app/vendor/craftcms/cms/src/elements/Category.php(631): craft\elements\Category->getGroup()
2024-10-22 09:23:50 #1 /app/vendor/craftcms/cms/src/base/Element.php(5397): craft\elements\Category->getFieldLayout()
2024-10-22 09:23:50 #2 /app/vendor/craftcms/cms/src/base/Element.php(2184): craft\base\Element->fieldByHandle('displayName')
2024-10-22 09:23:50 #3 /app/vendor/craftcms/cms/src/helpers/Template.php(137): craft\base\Element->__get('displayName')
2024-10-22 09:23:50 #4 /app/storage/runtime/compiled_templates/0d/0d889a6dde3e8bd53599c19572535ea3.php(210): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\elements\Category), 'displayName', Array, 'any', false, false, false, 87)
2024-10-22 09:23:50 #5 /app/vendor/twig/twig/src/Template.php(171): __TwigTemplate_6d286c71ea25c48a0ca88a9cb50145b8->block_content(Array, Array)
2024-10-22 09:23:50 #6 /app/storage/runtime/compiled_templates/74/74bb8e8021ea1e62dccbf4b9d35a8421.php(620): Twig\Template->displayBlock('content', Array, Array)
2024-10-22 09:23:50 #7 /app/vendor/twig/twig/src/Template.php(171): __TwigTemplate_8b11f7a9641fe828375af68f43bf809d->block_main(Array, Array)
2024-10-22 09:23:50 #8 /app/vendor/twig/twig/src/Template.php(134): Twig\Template->displayBlock('main', Array, Array, false)
2024-10-22 09:23:50 #9 /app/storage/runtime/compiled_templates/87/87955bbd77ac1fcd4a29732bea0af049.php(109): Twig\Template->displayParentBlock('main', Array, Array)
2024-10-22 09:23:50 #10 /app/vendor/twig/twig/src/Template.php(171): __TwigTemplate_7c9456f897483db3aa0bb7bcb83e3fce->block_main(Array, Array)
2024-10-22 09:23:50 #11 /app/storage/runtime/compiled_templates/74/74bb8e8021ea1e62dccbf4b9d35a8421.php(377): Twig\Template->displayBlock('main', Array, Array)
2024-10-22 09:23:50 #12 /app/vendor/twig/twig/src/Template.php(171): __TwigTemplate_8b11f7a9641fe828375af68f43bf809d->block_body(Array, Array)
2024-10-22 09:23:50 #13 /app/storage/runtime/compiled_templates/25/25ad2883e2f857d0748d42629b8b1441.php(94): Twig\Template->displayBlock('body', Array, Array)
2024-10-22 09:23:50 #14 /app/vendor/twig/twig/src/Template.php(394): __TwigTemplate_419c3a7e22a594d1a4e2354ef8277e3b->doDisplay(Array, Array)
2024-10-22 09:23:50 #15 /app/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
2024-10-22 09:23:50 #16 /app/storage/runtime/compiled_templates/0b/0bd978ccda7694be8ebf8f72244dff32.php(56): Twig\Template->display(Array, Array)
2024-10-22 09:23:50 #17 /app/vendor/twig/twig/src/Template.php(394): __TwigTemplate_b62d9557f21435ba24bd3bb93a6529d2->doDisplay(Array, Array)
2024-10-22 09:23:50 #18 /app/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
2024-10-22 09:23:50 #19 /app/storage/runtime/compiled_templates/74/74bb8e8021ea1e62dccbf4b9d35a8421.php(180): Twig\Template->display(Array, Array)
2024-10-22 09:23:50 #20 /app/vendor/twig/twig/src/Template.php(394): __TwigTemplate_8b11f7a9641fe828375af68f43bf809d->doDisplay(Array, Array)
2024-10-22 09:23:50 #21 /app/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
2024-10-22 09:23:50 #22 /app/storage/runtime/compiled_templates/87/87955bbd77ac1fcd4a29732bea0af049.php(56): Twig\Template->display(Array, Array)
2024-10-22 09:23:50 #23 /app/vendor/twig/twig/src/Template.php(394): __TwigTemplate_7c9456f897483db3aa0bb7bcb83e3fce->doDisplay(Array, Array)
2024-10-22 09:23:50 #24 /app/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
2024-10-22 09:23:50 #25 /app/storage/runtime/compiled_templates/66/66f905900ca65547dbd7f688c0ac2515.php(53): Twig\Template->display(Array, Array)
2024-10-22 09:23:50 #26 /app/vendor/twig/twig/src/Template.php(394): __TwigTemplate_c3a77e0687817c986e55c2bcf15250c8->doDisplay(Array, Array)
2024-10-22 09:23:50 #27 /app/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
2024-10-22 09:23:50 #28 /app/storage/runtime/compiled_templates/0d/0d889a6dde3e8bd53599c19572535ea3.php(62): Twig\Template->display(Array, Array)
2024-10-22 09:23:50 #29 /app/vendor/twig/twig/src/Template.php(394): __TwigTemplate_6d286c71ea25c48a0ca88a9cb50145b8->doDisplay(Array, Array)
2024-10-22 09:23:50 #30 /app/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
2024-10-22 09:23:50 #31 /app/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
2024-10-22 09:23:50 #32 /app/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\Template->render(Array)
2024-10-22 09:23:50 #33 /app/vendor/twig/twig/src/Environment.php(280): Twig\TemplateWrapper->render(Array)
2024-10-22 09:23:50 #34 /app/vendor/craftcms/cms/src/web/View.php(495): Twig\Environment->render('blitz/_utilitie...', Array)
2024-10-22 09:23:50 #35 /app/vendor/craftcms/cms/src/web/View.php(548): craft\web\View->renderTemplate('blitz/_utilitie...', Array)
2024-10-22 09:23:50 #36 /app/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(57): craft\web\View->renderPageTemplate('blitz/_utilitie...', Array, 'cp')
2024-10-22 09:23:50 #37 /app/vendor/yiisoft/yii2/web/Response.php(1109): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
2024-10-22 09:23:50 #38 /app/vendor/craftcms/cms/src/web/Response.php(338): yii\web\Response->prepare()
2024-10-22 09:23:50 #39 /app/vendor/yiisoft/yii2/web/Response.php(340): craft\web\Response->prepare()
2024-10-22 09:23:50 #40 /app/vendor/yiisoft/yii2/base/Application.php(390): yii\web\Response->send()
2024-10-22 09:23:50 #41 /app/web/index.php(45): yii\base\Application->run()
2024-10-22 09:23:50 #42 {main} {"memory":9395816,"exception":"[object] (yii\\base\\InvalidConfigException(code: 0): Category is missing its group ID at /app/vendor/craftcms/cms/src/elements/Category.php:737)"} 
2024-10-22 09:23:50 2024-10-22 08:23:50 [web.INFO] [application] Request context:
2024-10-22 09:23:50 $_GET = [
2024-10-22 09:23:50     'p' => 'admin/blitz/diagnostics/pages/page'
2024-10-22 09:23:50     'pageId' => '1012'
2024-10-22 09:23:50     'site' => 'default'
2024-10-22 09:23:50 ]
2024-10-22 09:23:50 $_POST = []
2024-10-22 09:23:50 $_FILES = []
2024-10-22 09:23:50 $_COOKIE = [
...
```

**Comment by bencroker** [comment]
> Category is missing its group ID

Hmm, could it be that the category in question has been deleted (or its group has)?

**Comment by webrgp** [comment]
`elementQueryType.type` returns just the `craft\elements\Category` string, so `create(elementQueryType.type)` is just creating a empty instance of Category. 

I agree that `displayName` should be working, but for some reason it ain't 🤔.

**Comment by webrgp** [comment]
Does adding the parenthesis on your end breaks it?

**Comment by bencroker** [comment]
No, that works too. Does your category have a custom field on it with the handle `displayName`, by any chance?

**Comment by bencroker** [comment]
That must have been it! Fixed in https://github.com/putyourlightson/craft-blitz/commit/caf1babf0ca159460e7465f445849af665815dde and released in [5.9.1](https://github.com/putyourlightson/craft-blitz/releases/tag/5.9.1) and [4.23.3](https://github.com/putyourlightson/craft-blitz/releases/tag/4.23.3). Thanks for reporting!

**Comment by webrgp** [comment]
FYI the site did have a `displayName` custom Plain Text field, but it's is used on a entry section. None of the category groups is using it (I checked both in CP and in the project config files).

Thanks for the fix!
